common/util/user_interaction.h

```diff
@@ -25,6 +25,19 @@ namespace term {
 // is an interactive session connected to a terminal. Otherwise just plain.
 std::ostream& bold(std::ostream& out, absl::string_view s);
 std::ostream& inverse(std::ostream& out, absl::string_view s);
+
+enum Color {
```

Comment:
Just out of curiosity, what would be most appropiate here?

```suggestion
enum class Color : uint8_t {
```

or

```suggestion
enum class Color {
```

Defaulting to 'int' underlying type to use the value as an index into an array seems weird to me, although I know the difference is insignificant in this case.

Reply:
I think the 'class' part is more important (type-safety), the fact that it internally is an int I would only worry when wrinting things for a microcontroller :)

Reply:
Right, makes sense. Sorry about the 'Suggested change' stuff, I thought it was just a visual thing, didn't realize it is mean to do a quick fix. The static cast was missing form the suggestion :facepalm: I'll fix it now

Reply:
Hah, and I thought it was the 'choose your own poison' button :)

I kindof like the suggestion change thing, how did you manage to include it ? Just adding a diff ?

Reply:
It shows in my web UI (probably experimental) but it should work for anyone by annotating a ``` block with sugestion (instead of c++ or whatever to get highlighting)